metadata.rb

```diff
@@ -26,4 +26,4 @@
 
 supports 'aix', '>= 6.1'
 
-chef_version '>= 12.1'
+chef_version '>= 12.5'
```

Comment:
You actually want to set this to 12.7 since 12.5 and 12.6 had some funky custom resource bugs.

Reply:
Also make sure to update the version in the readme